src/auth/SignInComponent.tsx

```diff
@@ -81,11 +81,11 @@ function SignInComponent() {
         <Grid container>
           <Grid size="grow">
             <Link href="#" variant="body2">
-              Forgot password?
+              {"Forgot password?"}
```

Comment:
This change is not necessary.

Reply:
I think it will be better this way, the change is small and the convention of the code in three places will be preserved